cl/recap/tasks.py

```diff
@@ -1429,12 +1434,56 @@ def mark_fq_status(fq, msg, status):
     fq.save()
 
 
+@app.task(bind=True, max_retries=5, interval_start=5 * 60)
+def send_docket_to_webhook(d_pk: int, since: datetime, epq_pk: int) -> None:
+    d = Docket.objects.get(pk=d_pk)
+    docket_entries = DocketEntry.objects.filter(
+        date_created__gte=since, docket=d
+    )
+    if docket_entries.count() == 0:
+        # No new docket entries.
+        return
+
+    epq = EmailProcessingQueue.objects.get(pk=epq_pk)
+    webhooks = Webhook.objects.filter(
+        event_type=WebhookEventType.RECAP_EMAIL,
+        user__profile__recap_email__in=epq.destination_emails,
+    )
+
+    serialized_docket_entries = []
+    for de in docket_entries:
+        serialized_docket_entries.append(DocketEntrySerializer(de).data)
+
+    for webhook in webhooks:
+        if webhook.enabled:
```

Comment:
Suggest flipping logic here to reduce nesting: `if not webhook.enabled: continue`

Reply:
Or, actually, even better, just filter out disabled webhooks in your DB query above with `enabled=True`.